test/Dialect/FIRRTL/parse-basic.fir

```diff
@@ -1689,12 +1689,15 @@ circuit BasicProps :
 
   ; CHECK-LABEL: module private @Path
   module Path :
-    ; CHECK-SAME: in %a: !firrtl.path
-    ; CHECK-SAME: out %b: !firrtl.path
-    input a : Path
-    output b : Path
-    ; CHECK: firrtl.propassign %b, %a : !firrtl.path
-    propassign b, a
+    ; CHECK-SAME: out %p: !firrtl.path
+    output p : Path
+
+    ; CHECK: %w = firrtl.wire sym @sym interesting_name : !firrtl.uint<8>
+    ; CHECK: %0 = firrtl.path <@Path::@sym>
+    ; CHECK: firrtl.propassign %p, %0 : !firrtl.path
+    wire w : UInt<8>
+    w is invalid
```

Comment:
Nit: for parsing the invalidation can be removed. Also, if you are using `FIRRTL version 3.0.0+` then this is `invalidate w`.